exercises/js/tabs.js

```diff
@@ -0,0 +1,48 @@
+function TabNavigator(div) {
+  this.div = div
+}
+
+TabNavigator.prototype.init = function() {
+  var modules = $(this.div);
+  modules.hide();
+  this.createTabs(modules);
+};
+
+TabNavigator.prototype.createTabs = function(modules) {
+  var modulesList = $('<div>');
+  modules.each(function() {
+    var $this = $(this);
+    var title = $this.find('h2').html();
+    var id = $this.attr('id');
+    modulesList.append('<li ' + 'data-id=' + id + '>' + title + '</li>');
+  });
+  modulesList.insertBefore(modules.first());
+  this.setModuleClickHandler(modules, modulesList);
+  this.selectedModule = modulesList.first();
+  this.selectedModuleId = this.selectedModule.attr('id');
+
+  // Open first tab
+  modulesList.children().first().trigger('click');
```

Comment:
Create a separate method for this and call it after calling `setModuleClickHandler` method.